### DISCUSSION TITLE
Making a matchmaking discord.py bot

### DISCUSSION THREAD

**Discussion by p4rthh** [Rapptz/discord.py#6236] [Q&A] [answered]
HOW CAN I MAKE A BOT THAT MAKES A NEW CHANNEL OR STARTS MATCHMAKING EACH TIME SOMEONE SAYS A COMMAND.
THANKS IN ADVANCE FOR WHOEVER HELPS ME

**Comment by Rapptz** [comment, marked as answer]
Hello.

This is not the place to ask for complete snippets or asking people to write your bot (or portions of it) for you. Users are expected to put in some effort such as code they've tried or a specific problem that would help them complete their implementation. Again, this is not the place to ask people to program things for you.

Thanks.
